apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html

```diff
@@ -0,0 +1,80 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+>
+    <tui-tabs [activeItemIndex]="activeTab">
+        @for (tab of tabs; track $index) {
+            <button
+                tuiTab
+                (click)="updateFades($index)"
+            >
+                <header tuiHeader>
+                    <h2
+                        tuiTitle
+                        [style.text-align]="'center'"
+                    >
+                        {{ tab }}
+                    </h2>
+                </header>
+            </button>
+        }
+    </tui-tabs>
+    <tui-scrollbar>
```

Comment:
Scrollbar disappears when switching between tabs. Try to put tui-scrollbar under the if{}